integration-tests/cypress/support/pages/functions/checkOperatorStatus.ts

```diff
@@ -0,0 +1,210 @@
+import { operatorNamespaces, operatorSubscriptions } from '../../constants';
+import {
+  createKnativeEventingUsingCLI,
+  createKnativeKafkaUsingCLI,
+  createKnativeServingUsingCLI,
+} from './knativeSubscriptions';
+
+// export const checkRedHatIntegrationCamelKOperatorStatus = (retries: number = 5) => {
```

Comment:
nit: can we remove the commented out part in all places?

Reply:
Sure @lokanandaprabhu